docs/installation.rst

```diff
@@ -1,80 +1,140 @@
 Installation
 ============
-These instructions will get the latest version of QuakeMigrate up and running.
+The QuakeMigrate is a predominantly Python package with some routines written and optimised in C. These are built and linked to QuakeMigrate at installation, which means you will need to ensure that there is a suitable compiler available (more details below).
+
+Supported operating systems
+---------------------------
+QuakeMigrate was developed and tested on Ubuntu 16.04/18.04, with the intention of being "platform agnostic". As of July 2020, the package has been successfully built and run on:
+
+- Ubuntu 16.04/18.04/20.04
+- Red Hat Enterprise Linux
+- Windows 10
+- MacOSX High Sierra 10.13.6
 
 Prerequisites
 -------------
-We recommend using Python environments to isolate the specific dependencies of QuakeMigrate.
+QuakeMigrate supports Python 3.6 or newer (3.7/3.8). We recommend using Anaconda as a package manager and environment management system to isolate and install the specific dependencies of QuakeMigrate. Instructions for downloading and installing Anaconda can be found `here <https://docs.anaconda.com/anaconda/install/>`_. If drive space is limited, consider using Miniconda instead, which ships with a minimal collection of useful packages.
 
 Setting up an environment
 *************************
-If you are using conda to manage your Python environments, you can use our quakemigrate.yml file to create and activate a minimally complete environment:
+Using conda, you can use our quakemigrate.yml file to create and activate a minimally complete environment:
 
 .. code-block:: bash
-
+    
     conda env create -f quakemigrate.yml
     conda activate quakemigrate
 
-Compiling C extensions
-**********************
+This will install the explicit dependencies of QuakeMigrate (as well as some additional sub-dependencies/useful packages). The full list of dependencies (and versions, where relevant) is:
+
+- matplotlib < 3.3
+- numpy
+- obspy >= 1.2
+- pandas >= 1
+- pyproj >= 2.6
+- scipy
+
+.. note:: These version pins are subject to change. We defer to ObsPy to select suitable versions for NumPy/SciPy.
+
+In addition, we use `NonLinLoc <http://alomax.free.fr/nlloc/>`_ and `scikit fmm <https://pythonhosted.org/scikit-fmm/>`_ as backends for producing 1-D traveltime lookup tables.
```

Comment:
This, for whatever reason, was cancelling the syntax highlighting that was saying this was a link in reStructured Text. Not sure if this would actually translate into an issue.